client/src/project/datasets/change/DatasetChange.container.js

```diff
@@ -65,7 +65,7 @@ function ChangeDataset(props) {
       titleField.help = datasetFormSchema.title.help;
       titleField.parseFun = undefined;
       image.value = {
-        options: [{ [Prop.URL]: props.dataset.mediaContent }],
+        options: [{ [Prop.URL]: props.dataset ? props.dataset.mediaContent : undefined }],
```

Comment:
`undefined` should only be used for variables that have not been assigned.

```suggestion
        options: [{ [Prop.URL]: props.dataset ? props.dataset.mediaContent : null }],
```